portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,13 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Makes sure user wants to return to previous page.
+ */
+function prevPage() {
```

Comment:
I added this because there are some alerts that come up when leaving a webpage without saving progress for example, so I just implemented to explore it a bit.